src/patternfly/components/Check/check.scss

```diff
@@ -18,17 +17,20 @@
   --#{$check}__label-required--FontSize: var(--#{$pf-global}--FontSize--sm);
   --#{$check}__label-required--Color: var(--#{$pf-global}--danger-color--100);
 
-  display: grid;
+  display: inline-grid;
```

Comment:
The check and radio components are expected to be block/full-width elements by default. I have a feeling this is going to have some other repercussions. For example, this will introduce a regression from this update - https://github.com/patternfly/patternfly/issues/5141

Why is it needed for them to be inline?